Formula/steem.rb

```diff
@@ -0,0 +1,33 @@
+class Steem < Formula
+  desc "Proof of Work blockchain with an unproven consensus algorithm."
+  homepage "https://steemit.com/"
+  url "https://github.com/steemit/steem.git",
+      :tag => "v0.8.5b",
```

Comment:
Ah, OK. But that's not in contradistinction to some stable version of the same software, just the developer being humble. I suppose this could go in devel only if the "beta" label is to be taken seriously?